layouts/_default/baseof.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
Why change this? This is one of the errors reported in the prettier run.

Reply:
When I run "npm run lint-fix". It changed it.

Reply:
@Yemaneberhan-Lemma, I updated npm dependencies last week. Can you run `yarn install` and then rerun `yarn lint-fix`?

Reply:
@hussainweb: Done.